packages/destination-actions/src/destinations/fullstory/__tests__/fullstory.test.ts

```diff
@@ -27,11 +27,9 @@ describe('FullStory', () => {
       const eventName = 'test-event'
 
       const properties = {
-        eventData: {
-          'first-property': 'first-value',
-          second_property: 'second_value',
-          thirdProperty: 'thirdValue'
-        },
+        'first-property': 'first-value',
```

Comment:
These properties should never have been nested in an `eventData` node in these tests. This bug in the tests was masked since the comparison was directly against the same `properties` variable value. It was exposed when property names changed due to type suffixing.